pkg/cluster/root_app.go

```diff
@@ -101,22 +111,11 @@ func ConstructRootApp(
 			Name:  fmt.Sprintf("tags.%s", subchartName),
 			Value: "true",
 		})
-		var ignoreDiffs []argoappv1.ResourceIgnoreDifferences
 		if cs.ClusterAutoscalerEnabled {
-			casSubchartName, err := clusterspec.ClusterAutoscalerSubchartNameFromClusterSpec(cs)
+			helmParams, ignoreDiffs, err = enableClusterAutoscaler(cs.ApiProvider, helmParams, ignoreDiffs)
```

Comment:
After this change, is `clusterspec.ClusterAutoscalerSubchartNameFromClusterSpec` still used anywhere? If not, delete it.

Reply:
I guess not, can be deleted

Reply:
It's being used in a test. We can proceed to remove the test and the function.